src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -1055,7 +1058,7 @@ namespace winrt::TerminalApp::implementation
 
     // Method Description:
     // - Reloads the settings from the settings.json file.
-    void AppLogic::_ReloadSettings()
+    void AppLogic::_ReloadSettings(const bool keybindingsOnly)

```

Comment:
nits...
- update method description
- I'd rename this `needRefreshUI` or something similar just to make it more general